README.md

```diff
@@ -333,6 +333,16 @@ When `enableLowInitialPlaylist` is set to true, it will be used to select
 the lowest bitrate playlist initially.  This helps to decrease playback start time.
 This setting is `false` by default.
 
+##### ignorePlayerSize
```

Comment:
In terms of naming, I think it might be a bit easier to use a positive and have it be true by default. Also maybe a bit more precise for rendition selection. Maybe something like `limitRenditionSelectionByPlayerDimensions`, though that name could be too long.

Reply:
Let's go with `limitRenditionByPlayerDimensions`, default to `true`

Reply:
I believe I have updated all of the things!